code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -2400,31 +2373,21 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_name = "glass of Darkbrew"
 	glass_desc = "A pitch black drink that's commonly confused with a type of coffee."
 
-/datum/reagent/consumable/ethanol/darkbrew/on_mob_life(mob/living/carbon/M)
-	if(isshadow(M))
-		quality = RACE_DRINK
-	else
-		M.adjust_disgust(25)
-	return ..()
+	species_required = "shadow"
 
-/datum/reagent/consumable/ethanol/hollow_bone
+/datum/reagent/consumable/ethanol/species_drink/hollow_bone
 	name = "Hollow Bone"
 	color = "#FCF7D4" //(252, 247, 212)
 	description = "Shockingly none-harmful mix of toxins and milk."
```

Comment:
non

i'm a useful member of the community

Reply:
thank